server/routes.go

```diff
@@ -497,6 +497,41 @@ func (b *BaseRouter) addActivityRoutes() {
 		}
 		c.AbortWithStatusJSON(http.StatusBadRequest, ErrorResponse{Error: err.Error()})
 	})
+
+	activity.PUT(":id", func(context *gin.Context) {
```

Comment:
We might want to rename `context` to `c` to keep it uniform with the rest of the handler funcs.

Same for delete below.

Reply:
I don't agree on this one. I think the others should change. Robert C. Martin has an entire chapter on naming in [Clean Code](https://www.goodreads.com/book/show/3735293-clean-code) which I believe to be the best take on the subject.

Reply:
@stephaje I personally prefer keeping it short as long as its still quickly understandable (unfortunately dont have 30 buckaroos for Robertos book to see his points).

But regardless of recommendations, in this case, it's probably for the best to stick with the shorter variant `c` so it's inline with the other handlers, instead of refactoring the whole file (routes.go is a chubster).